### PR TITLE
Fix invite link styling to distinguish from members list

### PR DESCRIPTION
**Pull request by ryanjduffy**
Fixes https://github.com/RecordReplay/customer-support/issues/111

After / Before

image

### CODE REVIEW

File: src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx [hunk 1]
```diff
@@ -107,7 +107,11 @@ const OrganizationSettings = ({ workspaceId }: { workspaceId: string }) => {
   const disabled = !workspace.isOrganization;
 
   return (
-    <div className={classNames("space-y-4", { "text-gray-500": disabled })}>
+    <div
+      className={classNames("-mr-4 max-h-full space-y-4 overflow-y-auto pr-4", {
```
Comment (ryanjduffy) [inline src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx:111]: Moved the scrolling support out of the general settings pane and into the pane that needs it. Other settings panes in personal settings already handle this correctly.

File: src/ui/components/shared/SettingsModal/SettingsNavigation.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .settings-modal nav {
-  padding: 36px;
+  padding: 32px;
```
Comment (ryanjduffy) [inline src/ui/components/shared/SettingsModal/SettingsNavigation.css:2]: adjusting padding to match tailwind sizes

**Review by jonbell-lot23 — APPROVED** [review]
👍

**Comment by jasonLaster** [comment]
Nice work!

**Comment by ryanjduffy** [comment]
Noticed that it didn't work for dark theme so I swapped out the bg color with a theme-based variable.

**Comment by jaril** [comment]
@ceceliacreates _maybe_ a docs change?

**Pull request merged**